__tests__/app.test.js

```diff
@@ -97,3 +97,52 @@ describe("GET /api/articles/:article_id, gets an article by its id", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments; adds a comment for an article by article id", () => {
+  test("posts a comment w/ a username and body", () => {
+    const newComment = { username: "rogersop", body: "great article!" }; //in my model do need to change username to author?
+
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        console.log(response, "response in TEST");
+        // expect(response.body.comment.author).toBe("rogersop");
+        expect(response.body.comment).toHaveProperty("author", "rogersop");
```

Comment:
.toMatchObject or .objectContaining are useful here to be thorough and check both keys and values